fmriprep/utils/bids.py

```diff
@@ -193,3 +193,52 @@ def _grp_echos(x):
     subj_data.update({"bold": ses_uids})
 
     return subj_data, layout
+
+
+def write_derivative_description(bids_dir, deriv_dir):
+    from fmriprep import __version__
+
+    desc = {
+        'Name': 'fMRIPrep output',
+        'BIDSVersion': '1.1.1',
+        'PipelineDescription': {
+            'Name': 'fMRIPrep',
+            'Version': __version__,
+            # We could parse the version to reference a commit or tag
+            'CodeURL': 'https://github.com/poldracklab/fmriprep',
```

Comment:
We can use the [`DOWNLOAD_URL`](https://github.com/poldracklab/fmriprep/blob/master/fmriprep/info.py#L49)

Reply:
Does that work for non-tagged versions?

Reply:
Yeah, of course not. But we could fall back to the repo in that case.

Reply:
But with versioneer we should be able to get a specific URL for all actual commits.

Reply:
Maybe I'm wrong, but the general case here is that the version is tagged.

I did not mean that the specific URL for non-tagged versions would not be best. Sure it is, and if versioneer provides it cheap, much better :). I proposed the `DOWNLOAD_URL` because it is already there for you to use in this PR without any complication.